src/test/java/org/candlepin/subscriptions/tally/billing/ContractsControllerTest.java

```diff
@@ -119,7 +119,7 @@ public void resetSingleton() throws Exception {
   void testThrowsIllegalStateExceptionWhenProductIsNotContractEnabled() throws Exception {
```

Comment:
The ContractsControllerTest still contains references to `uom` in:
- method `defaultUsage`
- method `createSubscriptionDefinition`